docs/UserGuide.md

```diff
@@ -513,9 +534,11 @@ To allow more flexibility in the input format, we have to sacrifice some validat
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
 2. **If you minimize the Help Window** and then run the `help` command (or use the `Help` menu, or the keyboard shortcut `F1`) again, the original Help Window will remain minimized, and no new Help Window will appear. The remedy is to manually restore the minimized Help Window.
-3. **There is an unused space below the command history box**. 
+3. If you type ``/`` in the input field for the add or edit command, apart from the input field for the name, it might cause the input to be wrongly parsed into other fields.
```

Comment:
I discussed with @MAOXIONGKAI and we feel it is not very wise put this in the known issues. Reason being we are exposing our own weaknesses to the testers and giving them free marks for grab (known issues provide no immunity)